features/steps/validation_handling.py

```diff
@@ -222,6 +222,17 @@ def handle_nested(instance):
 def is_list_of_tuples_or_none(var):
     return isinstance(var, list) and all(item is None or isinstance(item, tuple) for item in var)
 
+def select_final_instance(instances):
+    """"
+    The 'its final' predicament seems to be a statement that can be applied to either 
```

Comment:
```suggestion
    The 'its final' predicate seems to be a statement that can be applied to either 
```

Reply:
This line is no longer present after re-syncing with `development`.  However a search across files located it in another spot, which I have addressed in the most recent push.